test/unittests/libdevcrypto/LibSnark.cpp

```diff
@@ -338,6 +334,155 @@ BOOST_AUTO_TEST_CASE(pairingNullInput)
 	BOOST_CHECK(!r.first);
 }
 
+BOOST_AUTO_TEST_CASE(generateRandomPoints)
```

Comment:
Of it's actually testing SIMD operations, then probably better move it to separate file